### PR TITLE
Comments and did todo for commit on tradelist class

### PR DESCRIPTION
**Pull request by ctevans**

### CODE REVIEW

File: Skill/app/src/main/java/com/skilltradiez/skilltraderz/TradeList.java [hunk 1]
```diff
@@ -156,10 +179,40 @@ public boolean commit(UserDatabase userDB) {
...
+        /** There might be a bug here, I did this delete trade todo... /signed Cole **/
         newTrades.clear();
         for (ID tradeId : deletedTrades) {
-            //TODO Delete trade
+            Trade trade = DatabaseController.getTradeByID(tradeId);
+            User currentUser = DatabaseController.getAccountByUserID(getOwnerID());
+            User tradePartner = DatabaseController.getAccountByUserID(trade.getHalf2().getUser());
+
+            //If given a null just break through it.
+            if (tradeId == null){
```
Comment (nwoeanhinnogaehr) [inline Skill/app/src/main/java/com/skilltradiez/skilltraderz/TradeList.java]: if there's a null, it probably indicates a bigger error somewhere else. I don't think it should be ignored

Comment (ctevans) [inline Skill/app/src/main/java/com/skilltradiez/skilltraderz/TradeList.java]: I agree with this, I am trying to fit in an exception being thrown. Having a very bad struggle implementing it.

Comment (nwoeanhinnogaehr) [inline Skill/app/src/main/java/com/skilltradiez/skilltraderz/TradeList.java]: well if it's null, it'll just throw a nullpointerexception, which is probably fine

Comment (ctevans) [inline Skill/app/src/main/java/com/skilltradiez/skilltraderz/TradeList.java]: I'm not sure then what the point of the if statement is... if that is true. o_o

Comment (nwoeanhinnogaehr) [inline Skill/app/src/main/java/com/skilltradiez/skilltraderz/TradeList.java]: yeah I was trying to say you can just remove the whole thing

Comment (ctevans) [inline Skill/app/src/main/java/com/skilltradiez/skilltraderz/TradeList.java]: Alrighty, made the change and removed it and pushed to my origin master

**Pull request merged**